### PR TITLE
Add Waifu.im API to Anime

### PR DESCRIPTION
**Pull request by Bucolo**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Shikimori](https://shikimori.one/api/doc) | Anime discovery, tracking, forum, rates | `OAuth` | Yes | Unknown |
 | [Studio Ghibli](https://ghibliapi.herokuapp.com) | Resources from Studio Ghibli films | No | Yes | Yes |
 | [Trace Moe](https://soruly.github.io/trace.moe-api/#/) | A useful tool to get the exact scene of an anime from a screenshot | No | Yes | No |
+| [Waifu.im](https://waifu.im/docs) | Get waifu pictures from an archive of over 4000 images and multiple tags | No | Yes | No |
```
Comment (pawelborkar) [inline README.md:160]: The site is down. Please fix it and let me know

Comment (pawelborkar) [inline README.md:160]: The site is down. Please fix it and let me know

Comment (Bucolo) [reply README.md:160]: Yes, I was reworking it a bit and had to go to sleep before finishing, I'll let you know when it's back. Sorry for the inconvenience.

Comment (pawelborkar) [reply README.md:160]: sure, no problem.

Comment (Bucolo) [reply README.md:160]: It's normally back, sorry for the wait.

**Review by matheusfelipeog — APPROVED** [review]
Looks like it's all right around here now.

Thanks for your contribution @Bucolo ;)

**Pull request merged**